airflow/cli/commands/webserver_command.py

```diff
@@ -383,7 +383,7 @@ def webserver(args):
             "--workers",
             str(num_workers),
             "--worker-class",
-            str(args.workerclass),
+            "uvicorn.workers.UvicornWorker",
```

Comment:
Is this the only one supported worker after switch to the Connexion 3? If so we still need to get this value from the airflow configuration [`[webserver] worker_class`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-class).

It code be changed to default, and if user provide any other values, then replace it to the  `uvicorn.workers.UvicornWorker` with future warning, some current implementation could be found into the [`airflow.configuration.AirflowConfigParser.validate`](https://github.com/apache/airflow/blob/f36e1c75dc47dd29ad89d66ce357e333174cd29f/airflow/configuration.py#L717)
 
In additional newsfragment should be created with [description of significant changes](https://github.com/apache/airflow/tree/main/newsfragments)

Reply:
I believe this is just a temporary change to test it, but yes - we shoudl keep the option to be able to use original workers (here and in the internal_api). So we should leave that conversation open till then.

Reply:
>[Running the application](https://connexion.readthedocs.io/en/latest/v3.html#running-the-application)
There have been 2 changes related to running the application:
>
>You now MUST run the Connexion application instead of the underlying Flask application.
>You should use an ASGI server instead of a WSGI server.

So I guess there is no way to run Connexion 3 with default gunicorn workers because it is implements WSGI and `uvicorn.workers.UvicornWorker` adds ASGI support or it works as middleware, not sure. But maybe I miss something

Reply:
Well. Cool then we have to document that we change to uvicorn. We have no othe way I think and IMHO gunicorn worker configuration is really not something that we promised - so yes. significant description and changing to uvicorn should be documented and we should likely - when we get closer to merging that one, review the configuration and documentation

Reply:
I guess so, that it not something really braking/important. I could say even more we only check with `sync`, all other potentially doesn't work well, or we have to patch them (gevent). I think most of the user even doesn't noticed that

Reply:
Yep